gef.py

```diff
@@ -8861,9 +8877,11 @@ def do_invoke(self, argv):
             insnum_step = 1
 
         start_address = align_address(addr)
+        ref_address = align_address(ref_addr)
```

Comment:
Why create a new variable here with a similar name? Just do the align below when you assign `base_offset`

Reply:
sorry, that was an artifact from a previous implementation i tried